tools/release/release_utils.sh

```diff
@@ -66,6 +66,24 @@ function publishImages {
         echo "push image: $image"
         docker tag $image:$baseTag $image:$curTag
         retry 5 docker push $image:$curTag
+
+    done
+    docker logout
+}
+
+function publishPackages{
+    local curTag=$1
+    local baseTag=$2
+    local dockerHubUser=$3
+    local dockerHubPassword=$4
+    docker login ghcr.io -u $dockerHubUser -p $dockerHubPassword
+    local images="$(docker images --format "{{.Repository}}" --filter=reference='goharbor/*:'$baseTag'')"
```

Comment:
Remove this line, because now images are passed in by parameters.

Reply:
I've made all the required changes.

Reply:
@Nageshbansal I need some time to test this PR.